uisdk/src/main/res/values-de-rDE/strings.xml

```diff
@@ -142,6 +142,7 @@
     <plurals name="kh_uisdk_hours_plural">
         <item quantity="one">%d Stunde</item>
         <item quantity="other">%d Stunden</item>
+        <item quantity="zero"/>
```

Comment:
should be in translations? There is no "zero" plural in ios. If you want to hide label for "0 hour" case make it in code. If someone will provide translations for this key in future you will get "0 hour 35 minutes"